tools/pd-tso-bench/main.go

```diff
@@ -218,7 +218,7 @@ func newStats() *stats {
 func (s *stats) update(dur time.Duration) {
 	s.count++
 	s.totalDur += dur
-	s.latencyList = append(s.latencyList, dur.Milliseconds())
+	s.latencyList = append(s.latencyList, float64(dur.Nanoseconds())/1e6)
```

Comment:
Will the memory be insufficient?

Reply:
If we have 100000000 values, each of them takes 8 bytes to store, then the total memory usage will be 800MB. I think it's ok if the benchmark doesn't last too long, though the sorting later will be very time-consuming. I think some optimization is needed further.